lib/sgtn-client/core/config.rb

```diff
@@ -152,6 +164,24 @@ def logger
           Logging.logger
         end
 
+
+        def loader
+          @loader ||= begin
+            config = SgtnClient::Config.configurations[SgtnClient::Config.default_environment]
+
+            loaders = []
+            loaders << SgtnClient::TranslationLoader::SgtnServer.new if config['vip_server']
+            loaders << SgtnClient::TranslationLoader::LocalTranslation.new if config['translation_bundle']
```

Comment:
See factory pattern suggestion in https://github.com/Xiaochao8/singleton/pull/55/files

Reply:
applied